domains/wisard-gaming.is-not-a.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "",
+
+    "domain": "is-not-a.dev",
+    "subdomain": "wisard-gaming",
+
+    "owner": {
+        "repo": "https://github.com/ShadowGaming100"
+    },
+
+    "record": {
+        "NS": ["darwin.ns.cloudflare.com", "mallory.ns.cloudflare.com"]
```

Comment:
What is the reason for you needing to use NS records?

Reply:
> What is the reason for you needing to use NS records?

i have given the reason